modules/simple_forms_api/app/services/simple_forms_api/pdf_stamper.rb

```diff
@@ -101,7 +97,7 @@ def self.perform_multistamp(stamped_template_path, stamp_path)
       out_path = "#{Common::FileHelpers.random_file_path}.pdf"
       pdftk = PdfFill::Filler::PDF_FORMS
       pdftk.multistamp(stamped_template_path, stamp_path, out_path)
-      File.delete(stamped_template_path)
+      Common::FileHelpers.delete_file_if_exists(stamped_template_path)
```

Comment:
I'm hoping this will remove at least one class of error we've been seeing. Other errors around the next line (renaming the file) may persist but I'd rather tackle those separately.